sdk/handleError.js

```diff
@@ -10,30 +10,36 @@ const { maybeTranslateErrorCode } = require('binaris-pickle');
 
 const { version } = require('../package.json');
 
-class APIError extends Error {}
+class APIError extends Error {
+  constructor(message, requestIdString) {
+    super(`${requestIdString || ''}${message}`);
+  }
+}
 
 function validateResponse(response) {
+  const requestId = get(response, 'headers.x-binaris-request-id');
+  const requestIdString = requestId ? `RequestId: ${requestId}\n` : '';
```

Comment:
Why do this here and not in the APIError constructor? Just pass to APIError requestId